packages/form-js-viewer/test/spec/render/components/form-fields/Text.spec.js

```diff
@@ -46,6 +46,8 @@ describe('Text', function() {
 ## h2
 ### h3
 #### h4
+##### h5
+###### h6
```

Comment:
We need to adjust the `expect` part of this test as well: https://github.com/bpmn-io/form-js/pull/450/files#diff-f782522d93784395284f1b8e2c8b29c8ff07908fcb77815c9f527d9aa8222e0dR83

You can execute the tests locally as well via `npm run test` or `npm run dev -- '@bpmn-io/form-js-viewer`